src/TestFramework/Coverage/JUnit/TestFileDataAdder.php

```diff
@@ -33,71 +33,63 @@
 
 declare(strict_types=1);
 
-namespace Infection\TestFramework\Coverage\XmlReport;
+namespace Infection\TestFramework\Coverage\JUnit;
```

Comment:
I thought that move was done in another PR?

Reply:
It did. It is Git thinks this file has something to do with PhpUnitXmlCoverageFactory